ivadomed/utils.py

```diff
@@ -640,17 +640,18 @@ def threshold_predictions(predictions, thr=0.5):
     return thresholded_preds
 
 
-def segment_volume(fname_model, fname_model_metadata, fname_image, fname_roi=None):
+def segment_volume(folder_model, fname_image, fname_roi=None):
     """Segment an image.
 
-    Segment an image (fname_image) using a already trained model (fname_model) given its
-    training parameters (fname_model_metadata). If provided, a RegionOfInterest (fname_roi)
+    Segment an image (fname_image) using a already trained model given its
+    training parameters (both in folder_model). If provided, a RegionOfInterest (fname_roi)
     is used to crop the image prior to segment it.
 
     Args:
-        fname_model (string): model filename (.pt) to use.
-        fname_model_metadata (string): Configuration file filename (.json) used for the training,
-            see https://github.com/neuropoly/ivado-medical-imaging/wiki/configuration-file
+        folder_model (string): foldername which contains
+            (1) the model ('model.pt') to use
+            (2) its configuration file ('model_metadata.json') used for the training,
```

Comment:
I suggest to keep the same basename for the .json and .pt files (as done e.g. with the BIDS convention between the NIfTI binaries and the json metadata).
```suggestion
            (2) its configuration file ('model.json') used for the training,
```